contacts/models.py

```diff
@@ -11,6 +11,7 @@ class Contact(models.Model):
     name = models.CharField(max_length=50)
     email = models.EmailField(max_length=30)
     phone_number = models.CharField(validators=[phone_regex], max_length=20, blank=True)
+    contact_image = models.ImageField(upload_to='media/contacts_images', default='default.png')
```

Comment:
1. we don't need to mention "media/" here...it will be picked automatically by `media_url` mentioned in settings.
2. we need to update `<form></form>` in html tag to add `enctype="multipart/form-data"` to make sure it accept files as well. see https://www.w3schools.com/tags/att_form_enctype.asp
3. we need to update `self.form_class(request.POST)` in `contacts/views.py:28` to accept `request.FILES` as well.